src/Components/Films/Films.js

```diff
@@ -25,4 +26,12 @@ const Films = ({ films, showFeatured, handleSearchEntry, searchByWord, searchFie
     )
 }
 
-export default Films;
\ No newline at end of file
+export default Films;
+
+Films.propTypes = {
+    films: PropTypes.array,
+    showFeatured: PropTypes.func,
+    searchByWord: PropTypes.func,
+    searchField: PropTypes.string,
+    handleSearchEntry: PropTypes.func
+}
```

Comment:
is this allowed to be shortened to func? or does it have to be written out as function?